examples/lidar.py

```diff
@@ -0,0 +1,53 @@
+import datetime as dt
+import camille
+
+start_date = dt.datetime(2018, 1,1)
+end_date = dt.datetime.now()
+
+installation = 'HS2'
+data_root = '...'
+prefix =
+    f'{data_root}/processed/lidar/{installation}'
+windiris_root =
+    f'{data_root}/LiDAR_WindIris/Data/{installation}/real_time_data'
+bazefetcher_root =
+    f'{data_root}/bazefield_sensor_extraction/baze_taglist_all_gz'
+
+windiris = camille.source.windiris(windiris_root)
+bazefield = camille.source.bazefetcher(bazefetcher_root)
+pitch_tag = f'HYS-{installation}-MRUNacelle-PitchAngle'
+roll_tag = f'HYS-{installation}-MRUNacelle-rollAngle'
+
+date = start_date
+
+def trunc_date(date): dt.datetime(date.year, date.month, date.day)

```

Comment:
This doesn't return, which I guess was not on purpose.

Reply:
It was not

Reply:
Thumbs up.